sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.skill.OverviewBody', {
+    extend: 'Ext.container.Container',
+    xtype: 'slate-cbl-skill-overviewbody',
+    requires: [
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    items: [
+        {
+            reference: 'skillStatement',
+
+            xtype: 'component',
+            autoEl: 'p',
+            padding: '16 32 0'
+        },
+        {
+            reference: 'demonstrationsTable',
+
+            xtype: 'component',
+            minHeight: 200,
+            tpl: [
+                '<table class="skill-grid">',
+                    '<thead>',
+                        '<tr class="skill-grid-header-row">',
+                            '<th class="skill-grid-header skill-grid-demo-index">&nbsp;</th>',
+                            '<th class="skill-grid-header skill-grid-demo-date">Date</th>',
+                            '<th class="skill-grid-header skill-grid-demo-level">Level</th>',
+                            '<th class="skill-grid-header skill-grid-demo-experience">Experience</th>',
+                            '<th class="skill-grid-header skill-grid-demo-context">Context</th>',
+                            '<th class="skill-grid-header skill-grid-demo-task">Performance&nbsp;Task</th>',
+                            // '<th class="skill-grid-header skill-grid-demo-artifact">Artifact</th>',
```

Comment:
It can be deleted, we ended up finding the values too long to fit as cells, and ArtifactURL got moved down to the full-width expandy row below the main rows